CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# CHANGELOG
+
+## [Unreleased]
+
+### Overview
+
+- Upgrades cosmos-sdk to `v0.47.3` LSM fork created by persistence, including [barberry](https://forum.cosmos.network/t/cosmos-sdk-security-advisory-barberry/10825) security fix
+- Migrates to cometbft `v0.37.2`
+- Upgrades ibc-go to `v7.1.0` LSM fork including [huckleberry](https://forum.cosmos.network/t/ibc-security-advisory-huckleberry/10731) security fix
```

Comment:
I think "includes" things might go into sublist, because there are multiple things to mention.